### DISCUSSION TITLE
Add two classes CombinedGeometry and PathGeometry just like WPF

### DISCUSSION THREAD

**Discussion by huangfuhao** [unoplatform/uno#10278] [Q&A] [answered]
1.Add  Two Classes CombinedGeometry ,PathGeometry just like WPF
2.Support one Method  Combine
```
public static PathGeometry Combine(Geometry geometry1, Geometry geometry2, GeometryCombineMode mode, Transform transform, double tolerance, ToleranceType type)
{
            return PathGeometry.InternalCombine(geometry1, geometry2, mode, transform, tolerance, type);
 }
```

![image](https://user-images.githubusercontent.com/116789821/198241627-20f3e428-b4be-412e-aaec-fcc2e68b0bb3.png)

![image](https://user-images.githubusercontent.com/116789821/198240781-25b3aa04-7cc6-4485-87c9-13ca3d84ca98.png)
****

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the suggestion! 

At this time, we cannot add this particular class because it is not supported by WinUI, mainly because the `Clip` Property is a [`RectangleGeometry`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.clip?view=winrt-22621#windows-ui-xaml-uielement-clip). That being said, on Windows (and only Skia on Uno Platform at this time), it's possible to use the composition API to do visual clipping with [`CompositionGeometricClip`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.composition.compositiongeometricclip?view=winrt-22621).

**Reply by huangfuhao** [reply]
Well, thank you，i will try it,
Thank you for your contribution!
